recipes/sphinxcontrib-programoutput/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "sphinxcontrib-programoutput" %}
+{% set version = "0.11" %}
+{% set sha256 = "cbec3ee1c3abd09e105115ab69cb5ade8ca1be9811565a844f973e93e0314837" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - sphinx
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
+
+  run:
+    - python
+    - six
+    - jinja2
+    - pygments
+    - docutils
+    - snowballstemmer
+    - babel
+    - alabaster
+    - imagesize
+    - requests
+    - sphinxcontrib-websupport
+    - colorama
+    - markupsafe
+    - pytz
+    - chardet
+    - idna
+    - urllib3
+    - certifi
+
+test:
+  imports:
+    - sphinxcontrib.programoutput
+
+about:
+  home: https://bitbucket.org/birkenfeld/sphinx-contrib
+  license: BSD 2-clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Sphinx extension to include program output'
+
+  description: |
+    A Sphinx extension to literally insert the output of arbitrary commands into documents, helping you to keep your command examples up to date.
```

Comment:
Please break this up over multiple lines

Reply:
Done